src/runtime/typemanager.cs

```diff
@@ -739,13 +835,34 @@ internal static void InitializeSlots(IntPtr type, Type impl)
                 // load them into a separate code page that is leaked
                 // intentionally.
                 InitializeNativeCodePage();
-                ret1 = NativeCodePage + native.Return1;
-                ret0 = NativeCodePage + native.Return0;
+                IntPtr ret1 = NativeCodePage + native.Return1;
+                IntPtr ret0 = NativeCodePage + native.Return0;
+                InitializeSlot(type, ret0, "tp_traverse", false);
+                InitializeSlot(type, ret0, "tp_clear", false);
```

Comment:
NIT: can you add parameter name to `false` here? (e.g. `paramName: false`) Otherwise it is unclear what it means.